packages/orchestrator-ui-components/src/pages/startPage/WfoStartPage.tsx

```diff
@@ -259,12 +254,11 @@ const taskListSummaryQueryVariables: GraphqlQueryVariables<Process> = {
     ],
 };
 
-const productsSummaryQueryVariables: GraphqlQueryVariables<ProductDefinition> =
-    {
-        first: 1000,
-        after: 0,
-        sortBy: {
-            field: 'name',
-            order: SortOrder.ASC,
-        },
-    };
+const productsSummaryQueryVariables: GraphqlQueryVariables<ProductsSummary> = {
+    first: 1000,
```

Comment:
I use 10000 here. It's an absurd number by design. I think this should be solved in the backed

Reply:
yeah i introduced a constant in this PR: [packages/orchestrator-ui-components/src/pages/startPage/WfoStartPage.tsx](https://github.com/workfloworchestrator/orchestrator-ui-library/pull/902/files#diff-3e4ee0aec957a400f858ee7e47fc03a09cd1aaeaec340328b2dc7feee6ef5cee) -- will apply that after merging that PR